translations/admin_ext.fr.yaml

```diff
@@ -174,7 +174,7 @@ all_roles: "Tous les r\xF4les"
 add_role: "Ajouter un r\xF4le"
 role_creation_error: "Le r\xF4le n'a pas \xE9t\xE9 cr\xE9\xE9"
 workspaces: Zone de travail
-Username: Login
+Username: Nom d'utilisateur
```

Comment:
@mcassier31 Is it right that username can be translated to `login` in French?

Reply:
@kingjia90 Not sure you still need this, but "username" is "nom d'utilisateur" :)

Reply:
yes, that helped, thank you! 
I've changed it directly on [POEditor](https://poeditor.com/join/project/XliCYYgILb) instead of here